src/falcor/transforms/index.js

```diff
@@ -196,3 +196,30 @@ export function remove ( collection, user, _id ) {
     });
 }
 
+export function archiveNode(nodeLabel, nodeId, userId) {
+  const query = `
+    MATCH (node:${nodeLabel} {_id: {nodeId} }) 
+    OPTIONAL MATCH (node)<-[rel]-() 
+    SET node.archived = true, 
+    node.archived_at = timestamp(), 
+    node.archiver = {userId}, 
+    rel.archived = true, 
+    rel.archived_at = timestamp(), 
+    rel.archiver = {userId} 
+    RETURN DISTINCT node as node`
```

Comment:
We needn't do `node as node` here since the default name will be whatever it started with, which in this case is also `node`.